### ISSUE TITLE
Alt text in dark + monochrome mode

### ISSUE THREAD

**Issue by xbarbie** [processing/p5.js-website#468]
### Most appropriate sections of the p5.js website?

Home

### What is your operating system?

Windows

### Web browser and version

Firefox 128.0 (20240704121409)

### Actual Behavior

Alt text does has white color, while the rectangle has a very light gray (almost white) background color.
This issue is happening when the dark mode and monochrome modes are both selected at the same time, see the screenshots below.

![alt_text_bug](https://github.com/user-attachments/assets/7f9f7c2d-8781-469b-8e4b-1c2fbcfe7326)

### Expected Behavior

Black background color for the rectangle and white color for the text. Or the opposite, black text on white background.

### Would you like to work on the issue?

no

**Comment by Qianqianye** [comment]
Thanks @xbarbie. I noticed this issue the other day as well, and I'm tagging the website designers @stefanietam @tyler-yin to take a look at this issue about alt text in dark + monochrome mode.

**Comment by AaratiAkkapeddi** [comment]
Hi! Happy to help with this when there is a decided behavior.

**Comment by tyler-yin** [comment]
Thank you for documenting this issue. Either option (black text on white background OR white text on black background) are acceptable fixes to this. For consistency with alt text + dark mode, I recommend black text on white background.

**Comment by Qianqianye** [comment]
Thank you all @AaratiAkkapeddi @tyler-yin @xbarbie! This is now resolved.
